student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, classroom:, **rest)
```

Comment:
- [ ] Please fix this typo. It should be `classroom`. Kindly remove the colon `:`.

Reply:
fix thanks

Reply:
Fixed.